cloverage/test/cloverage/coverage_test.clj

```diff
@@ -197,15 +199,21 @@
 (t/deftest test-instrument-gets-lines
   (inst/instrument #'cov/track-coverage
                    'cloverage.sample.exercise-instrumentation)
-  (let [cov @cov/*covered*
-        found (find-form cov '(+ 40 2))]
-    #_(with-out-writer "out/foo"
-        (doseq [form-info cov]
-          (println form-info)))
-    #_(println "Form is")
-    (t/is found)
-    (t/is (:line found))
-    (t/is (find-form cov '(inc (m c 0))))))
+  (let [cov @cov/*covered*]
+    (doseq [[form expanded] '{(+ 40)      (+ 40)
+                              (+ 40 2)    (. clojure.lang.Numbers (add (cloverage.instrument/wrapm
+                                                                        cloverage.coverage/track-coverage 10 40)
+                                                                       (cloverage.instrument/wrapm
+                                                                        cloverage.coverage/track-coverage 10 2)))
+                              (str 1 2 3) (str 1 2 3)
+                              (inc m c 0) (. clojure.lang.Numbers (inc (cloverage.instrument/wrapm
+                                                                        cloverage.coverage/track-coverage
+                                                                        101
+                                                                        (m c 0))))}]
+      (t/testing (format "Form %s (expanded to %s) should get instrumented" (pr-str form) (pr-str expanded))
+        (t/is (find-form cov expanded))
+        (let [found (find-form cov expanded)]
+          (t/is (:line found)))))))
```

Comment:
Since `(+ 40 2)` is actually replaced with the inlined version now I had to rework this test. I kept the original two forms being checked and added a few more that aren't replaced with inlined versions. I put them all in a `doseq` form so I could add `testing` context which makes it a lot easier to understand which assertions are failing when the test fails